contracts/margin/external/ERC20/ERC20Position.sol

```diff
@@ -488,17 +477,39 @@ contract ERC20Position is
             "ERC20Position#close: Cannot close 0 amount"
         );
 
-        assert(tokenAmount <= balance);
```

Comment:
did you just remove this because safemath will catch it?

Reply:
yup